Content.Server/GameObjects/Components/Morgue/CrematoriumEntityStorageComponent.cs

```diff
@@ -92,7 +111,34 @@ public void Cremate()
                 TryOpenStorage(Owner);
 
                 EntitySystem.Get<AudioSystem>().PlayFromEntity("/Audio/Machines/ding.ogg", Owner);
-            });
+            }, _cremateCancelToken.Token);
+        }
+
+        public SuicideKind Suicide(IEntity victim, IChatManager chat)
+        {
+
+            if (victim.TryGetComponent<IBody>(out var body))
+            {
+                foreach (var part in body.Parts.Values)
+                {
+                    if (!body.TryDropPart(part, out var dropped))
+                    {
+                        continue;
+                    }
+
+                    foreach (var drop in dropped)
+                    {
+                        Contents.Insert(drop.Owner);
+                    }
+                }
+            }
```

Comment:
Paging doctorbodyleaf @DrSmugleaf 

It's probably better to make a gibbing helper and just have anything that needs to gib call it as you also re-used this code else where. Even though it'll be barebones for now when we get better gibbing it's just 1 spot to update.

Reply:
Maybe just chuck a TODO: gibbing on the 3 spots and we can sort it out later?

Reply:
Added!